cmd/config.go

```diff
@@ -80,7 +80,8 @@ type Config struct {
 	TimeEvents            bool
 
 	// State Syncer
-	StateChangeDir string
+	StateChangeDir               string
+	NumMempoolTxStateSyncerLimit uint64
```

Comment:
Generally a good practice to prefix config names with the part of the system they apply to
```suggestion
	StateSyncerMempoolTxnSyncLimit uint64
```